src/subscription_manager/managerlib.py

```diff
@@ -92,20 +106,22 @@ def persist_consumer_cert(consumerinfo):
 
 
 class CertificateFetchError(Exception):
-    def __init__(self, errors):
+    def __init__(self, errors: Iterable[Exception]):
         self.errors = errors
 
-    def __str__(self, reason=""):
+    def __str__(self, reason: str = "") -> str:
+        # FIXME Explicitly convert errors to strings
         msg = "Entitlement Certificate(s) update failed due to the following reasons:\n" + "\n".join(
             self.errors
         )
         return msg
 
 
-def fetch_certificates(certlib):
+# FIXME Does not seem to be used
+def fetch_certificates(certlib) -> Literal[True]:
```

Comment:
this was used only by the GUI code; can you please do a PR before this dropping this dead code? IMHO it'd be better than type-hinting it...